api/ca.proto

```diff
@@ -34,11 +34,15 @@ message NodeCertificateStatusResponse {
 }
 
 message IssueNodeCertificateRequest {
-	NodeRole role = 1;
+	// DEPRECATED: Role is now selected based on which secret is matched.
+	NodeRole role = 1 [deprecated=true];
```

Comment:
Isn't it a thing with protobuf that you can drop number fields as long as you don't reuse numbers because when the newer version unmarshals older protobufs, it'll just ignore those fields? Does that not apply in this case?